scinet/scinet.py

```diff
@@ -50,11 +50,20 @@ def __init__(self, hyp):
         self.leadingLoss = hyp.leadingLoss(reduction='sum')
         self.lossFunct = self._VAE_loss
 
+        self.trainCounter = 0
+        self.annealEpoch = hyp.annealEpoch
+        self.annealWeight = self._update_anneal_weight()
+
+    def _update_anneal_weight(self):
+        if self.annealEpoch == 0:
+            return 1
+        return min(1, self.trainCounter / self.annealEpoch)
+
```

Comment:
Actually, sorry, but this is the perfect chance for using a `@property` which I fuckin love, I couldn't resist changing it. It has no other effect, I just like them.